docs/reference/search/aggregations/bucket/sampler-aggregation.asciidoc

```diff
@@ -0,0 +1,160 @@
+[[search-aggregations-bucket-sampler-aggregation]]
+=== Sampler Aggregation
```

Comment:
Not sure if it already is, but can't see it if so: we should mark this feature as experimental in the docs